src/ayb_db/models.rs

```diff
@@ -175,20 +175,22 @@ pub struct Entity {
 
 #[derive(Debug, Serialize, Deserialize)]
 pub struct PartialEntity {
-    pub slug: Option<String>,
-    pub entity_type: Option<i16>,
     pub display_name: Option<Option<String>>,
```

Comment:
While I prefer the single level of Option without needing to encode "no update," (using a map as you explained), if we keep it this way (it's ok if you do), copy some of the text from [this comment](https://github.com/marcua/ayb/pull/248#discussion_r1439326967) as a code comment so readers can understand how it works.